rclrs_tests/src/lib.rs

```diff
@@ -1,5 +1,8 @@
 #![cfg(test)]
 
 mod client_service_tests;
+// Disabled in Foxy due to https://github.com/ros2/rosidl/issues/598
+#[cfg(all(not(ros_distro = "foxy"), not(ros_distro = "galactic")))]
```

Comment:
Why was this disabled for Galactic? It seemed that only Foxy was affected

Reply:
No, actually also Galactic. I was wrong at first – should correct the comment.

Reply:
As proof, I removed the galactic part temporarily: https://github.com/ros2-rust/ros2_rust/actions/runs/3452502455/jobs/5762451384

Reply:
In that case, can you update the comment to mention `galactic` as well, instead of only `foxy`? Thanks

Reply:
@nnmm can you update the comment here? I can push the change, but given that this PR is part of a bigger PR, I thought of leaving it to you.